python/lsst/validate/drp/calcSrd.py

```diff
@@ -389,12 +443,20 @@ def calcAMx(groupView, D=5, width=2, magRange=None):
     magRange : 2-element list or tuple
         brighter, fainter limits of the magnitude range to include.
         E.g., `magRange=[17.5, 21.0]`
+    x : int
+        Which of AM1, AM2, AM3.  One of [1,2,3].
+    level : str
+        One of "minimum", "design", "stretch" indicating the level of the specification desired.
 
     Returns
     -------
-    list, list, list
+    pipeBase.Struct
         rmsDistMas, annulus, magRange
 
+    Raises
+    ------
+    ValidateError if `x` isn't in `getAstrometricSpec` values of [1,2,3]
```

Comment:
The usual convention for Python is for a bad argument to raise a `ValueError`. If we are going to retain a per-package exception tree then I feel this should throw a `ValidateValueError` that inherits from `ValueError` and `ValidateError`.